### PR TITLE
build: Use 1password secrets for most push secrets instead of github secrets

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

It's really difficult to manage our secrets in github actions secrets

## How This PR Solves The Issue

Manage most secrets in our team 1password account, and let Github Actions pull from that.

## Manual Testing Instructions

Review build from here and where it was tried out for
* [release](https://github.com/ddev-test/ddev/actions/runs/10856961949)
* [commit](https://github.com/ddev-test/ddev/actions/runs/10856949449)
* [push tagged image](https://github.com/ddev-test/ddev/actions/runs/10853648361)
* [push tagged dbimage](https://github.com/ddev-test/ddev/actions/runs/10853653152)

Most of the drama and learning was on 
* https://github.com/ddev-test/ddev/pull/5

And followup commits that show here and in https://github.com/ddev-test/ddev/commits/master/

### CODE REVIEW

File: docs/content/developers/release-management.md [hunk 1]
```diff
@@ -21,29 +21,36 @@ search:
...
-The following “Repository secret” environment variables must be added to <https://github.com/ddev/ddev/settings/secrets/actions>:
+These are normally configured in the repository environment variables.
 
-* `AMPLITUDE_API_KEY`: Key that enables Amplitude reporting. Environment variable for Make is `AmplitudeAPIKey`.
-* `AMPLITUDE_API_KEY_DEV`: Key that enables Amplitude reporting for development versions e.g. a PR build. Environment variable for Make is `AmplitudeAPIKey`.
 * `AUR_EDGE_GIT_URL`: The Git URL for AUR edge (normally `ddev-edge-bin`), for example `ssh://aur@aur.archlinux.org/ddev-edge-bin.git`.
 * `AUR_STABLE_GIT_URL`: The Git URL for AUR stable (normally `ddev-bin`), for example `ssh://aur@aur.archlinux.org/ddev-bin.git`.
+* `DOCKERHUB_USERNAME`: Username for pushing to `hub.docker.com` or updating image descriptions.
+* `FURY_ACCOUNT`: [Gemfury](https://gemfury.com) account that receives package pushes.
+* `HOMEBREW_EDGE_REPOSITORY`: Like `ddev/homebrew-ddev-edge` but might be another repo like be `ddev-test/homebrew-ddev-edge`.
```
Comment (stasadev) [inline docs/content/developers/release-management.md:32]:
```suggestion
* `HOMEBREW_EDGE_REPOSITORY`: Like `ddev/homebrew-ddev-edge` but might be another repository like be `ddev-test/homebrew-ddev-edge`.
```

File: docs/content/developers/release-management.md [hunk 2]
```diff
@@ -21,29 +21,36 @@ search:
...
+* `HOMEBREW_STABLE_REPOSITORY`: Like `ddev/homebrew-ddev` but might be another repo like `ddev-test/homebrew-ddev`.
```
Comment (stasadev) [inline docs/content/developers/release-management.md:33]:
```suggestion
* `HOMEBREW_STABLE_REPOSITORY`: Like `ddev/homebrew-ddev` but might be another repository like `ddev-test/homebrew-ddev`.
```

**Review by stasadev — APPROVED** [review]
Looks good to me.

**Review by stasadev — APPROVED** [review]

**Comment by rfay** [comment]
There are a couple of "auto" pushes that don't push on prerelease, mostly because we don't have a target for them. So here AUR and Fury pushes didn't get manually tested.
